js/main.js

```diff
@@ -1,16 +1,101 @@
-const getRandomInt = (firstValue, secondValue) => {
-  firstValue = Math.ceil(firstValue);
-  secondValue = Math.floor(secondValue);
-  if (firstValue < 0 || secondValue < 0 || (secondValue - firstValue) < 0) {return null;}
-  return Math.floor(Math.random() * (secondValue - firstValue + 1)) + firstValue;
+// const getRandomInt = (firstValue, secondValue) => {
+//   firstValue = Math.ceil(firstValue);
+//   secondValue = Math.floor(secondValue);
+//   if (firstValue < 0 || secondValue < 0 || (secondValue - firstValue) < 0) {return null;}
+//   return Math.floor(Math.random() * (secondValue - firstValue + 1)) + firstValue;
+// };
+// const getRandomGeo = (firstValue, secondValue, num) => {
+//   if (firstValue < 0 || secondValue < 0 || (secondValue - firstValue) < 0) {return null;}
+//   if (firstValue.toFixed(num) === secondValue.toFixed(num)) {return firstValue.toFixed(num);}
+//   return ((Math.random() * (secondValue - firstValue)) + firstValue).toFixed(num);
+// };
+
+const ADS_NUMBERS = 10;
+const ADS_TYPES = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+const CHECK_IN = ['12:00', '13:00', '14:00'];
+const CHECK_OUT = ['12:00', '13:00', '14:00'];
+const ADS_TITLES = [
+  'Просторное помещение',
+  'Тихое, уютное место для двоих',
+  'Светлое, просторное место',
+  'Всё по фен шую, как вы ждали',
+  'Вам захочется приехать сюда ещё раз'
+];
+const ADS_FEATURES = ['wifi', 'dishwasher', 'parking', 'elevator', 'conditioner'];
+const ADS_DISCRIPTION = [
```

Comment:
```suggestion
const ADS_DISCRIPTIONS = [
```
массивы должны быть во множественном числе